### PR TITLE
machine_configuration: Update README.md

### PR DESCRIPTION
**Pull request by hershpa**
Improving readme quality for clarity.
Signed-off-by: Hersh Pathak <hersh.pathak@intel.com>

### CODE REVIEW

File: machine_configuration/README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Ensure `intel-dgpu` MachineConfigPool is present.
 # Disable conflicting driver
 Run the command shown below to disable the loading of a potential conflicting driver, such as `ast` driver.
 
-**Note**: The `i915` driver depends on a ported `drm` module. Some other drivers, such as ast that depends on in-tree drm module might have a compatibility issues, known issue will be resolved on i915 driver for RHEL `9.x`, which will be used for RHOCP `4.13`. 
+**Note**: The `i915` driver depends on a ported `drm` module. Some other drivers, such as ast that depends on in-tree drm module might have compatibility issues. The known issue will be resolved on i915 driver for RHEL `9.x`, which will be used for RHOCP `4.13`.
```
Comment (uMartinXu) [inline machine_configuration/README.md:76]: do you think "have a compatibility issue" is better? :-)

Comment (hershpa) [reply machine_configuration/README.md]: Yes, that is perfect, updated PR.

**Comment by hershpa** [comment]
Any other feedback for this PR?

**Comment by uMartinXu** [comment]
It looks good to me!

**Pull request merged**